lib/perl/Genome/Command/DelegatesToResult.pm

```diff
@@ -63,12 +68,15 @@ sub input_hash {
 sub _input_hash {
     my $self = shift;
 
-    my $result_users = {
-        sponsor   => $self->sponsor // Genome::Sys->current_user,
-        requestor => $self->requestor,
-    };
-    if($self->label and $self->user) {
-        $result_users->{$self->label} = $self->user;
+    my $result_users = $self->result_users;
+    unless ($result_users) {
+        my $result_users = {
```

Comment:
probably don't want the 'my' here since you want to use the value outside the scope of this unless block